tests/main/lxd/task.yaml

```diff
@@ -0,0 +1,34 @@
+summary: Ensure that lxd works
+
+execute: |
+    wait_for_lxd(){
+        while ! printf "GET / HTTP/1.0\n\n" | nc -U /var/snap/lxd/common/lxd/unix.socket | MATCH "200 OK"; do sleep 1; done
+    }
+
+    echo "Install lxd"
+    snap install lxd
+
+    echo "Create a trivial container using the lxd snap"
+    wait_for_lxd
+    lxd init --auto
+    lxd.lxc launch ubuntu:16.04 my-ubuntu
```

Comment:
Yes, it will download an image and the image is big(ish). So I'm happy about any suggestions that can improve the speed here :)